idseq_dag/steps/generate_coverage_viz.py

```diff
@@ -36,6 +38,10 @@ def run(self):
         min_contig_size = self.additional_attributes.get("min_contig_size", MIN_CONTIG_SIZE)
 
         # Info DB contains the name and sequence length of each accession.
+        # IMPORTANT NOTE: This info_db file is currently nt_info.sqlite3.
+        # SQLite is known to fail intermittently in our pipeline when it is
+        # run inside of multiprocessing. So don't do that here, or else switch
+        # to berkeley DB, as detailed in https://jira.czi.team/browse/IDSEQ-1536.
```

Comment:
@morsecodist Is this cautionary comment still relevant, in view of your recent revert?

@gregdingle  Thanks for the note.  My understanding is that opening the DB in a subprocess, rather than the parent process, is actually marginally safer.  Difficult to argue decisively either way since the actual cause of the deadlock is likely a thread lock that's part of CPython or sqlite3 (my best guess, after pulling out lots hair and finding nothing truly definitive :).